src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.lang.reflect.Array;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Application {
+    FigureSupplier figureSupplier = new FigureSupplier();
+    public static void main(String[] qwe) {
+        Figure [] figure = {new Square(),new Rectangle(),new RightTriangle(),
```

Comment:
The array should be named with a plural noun, such as 'figures' instead of 'figure', to represent that it contains multiple elements.